Server/LauncherClient/Launcher.cs

```diff
@@ -122,8 +125,16 @@ private void game_start_timer_Tick(object sender, EventArgs e)
         public void SetConfigValue(string key, string value)
         {
             Configuration configuration = ConfigurationManager.OpenExeConfiguration(ConfigurationUserLevel.None);
+
+            if(key == "Secret")
+            {
+                //string encryptedValue = encryption.Encrypt(value);
+                string encryptedValue = MachineKeyEncryption.Protect(value, $"Secret for computer {computerKey}");
+                value = encryptedValue;
+            }
+
             configuration.AppSettings.Settings[key].Value = value;
-            configuration.AppSettings.SectionInformation.ProtectSection(null);
+            //configuration.AppSettings.SectionInformation.ProtectSection(null);
```

Comment:
I promise I'm gonna stop leaving this comment after this, but...

did you mean to leave this comment in the code? 

please keep thinking about it as you look at this PR in total, even though I've stopped commenting.